.pipelines/templates/PackageBuild.yml

```diff
@@ -37,6 +37,14 @@ parameters:
     # - name: build-artifacts
     #   rpmsTarball: cache.tar.gz
 
+  - name: isAllowToolchainRebuilds
+    type: string
+    default: "default"
```

Comment:
Why not default to true or false? and how is the value of "default" handled? (it seems to be skipped by the if/elif case below)

Reply:
With "default" I can pass no arguments to the toolkit and let toolkit's defaults take over. If I forced "true" or "false" here, then I'd need to keep up with every update of the defaults in the toolkit.